odoo/tools/misc.py

```diff
@@ -1383,22 +1383,37 @@ def format_datetime(env, value, tz=False, dt_format='medium', lang_code=False):
 def format_time(env, value, tz=False, time_format='medium', lang_code=False):
     """ Format the given time (hour, minute and second) with the current user preference (language, format, ...)
 
-        :param value: the time to format
-        :type value: `datetime.time` instance. Could be timezoned to display tzinfo according to format (e.i.: 'full' format)
-        :param format: one of “full”, “long”, “medium”, or “short”, or a custom date/time pattern
-        :param lang_code: ISO
+        :param {str, datetime, datetime.time} value: the time to format
+        :param {str} tz: name of the timezone  in which the given datetime should be localized
+        :param {str} time_format: one of “full”, “long”, “medium”, or “short”, or a custom date/time pattern
+        :param {str} lang_code: ISO
```

Comment:
Info fields don't use braces, with the possible exceptions of set literals.

For non-trivial types, the original `:type` is the correct version. For trivial types, they should be written out "bare" e.g.

    :param str lan_code: ...

Reply:
Done, I switched back to the original version.